Containerfile.mce.rhtap

```diff
@@ -0,0 +1,38 @@
+FROM registry.redhat.io/ubi8/nodejs-18:1 AS builder
+
+WORKDIR /app
+
+USER root
+ENV NPM_CONFIG_NODEDIR=/usr
+
+# Running installs concurrently fails on aarch64
+RUN npm ci --no-optional  --unsafe-perm --ignore-scripts
+RUN cd backend && npm ci --no-optional  --unsafe-perm
+RUN cd frontend && npm ci --no-optional --legacy-peer-deps --unsafe-perm
+RUN npm run build:backend
+RUN cd frontend && npm run build:plugin:mce
+
+# Remove build-time dependencies before packaging
+RUN cd backend && npm ci --no-optional --only=production --unsafe-perm
+
+FROM brew.registry.redhat.io/rh-osbs/rhacm2-nodejs-parent:rhel_8_nodejs_18
+
+WORKDIR /app
+ENV NODE_ENV production
+COPY --from=builder /app/backend/node_modules ./node_modules
+COPY --from=builder /app/backend/backend.mjs ./
+COPY --from=builder /app/frontend/plugins/mce/dist ./public/plugin
+USER 1001
+CMD ["node", "backend.mjs"]
+
+LABEL com.redhat.component="multicluster-engine-console-mce-container" \
+      name="multicluster-engine/console-mce-rhel8" \
+      summary="multicluster-engine-console-mce" \
+      io.openshift.expose-services="" \
+      io.openshift.tags="data,images" \
+      io.k8s.display-name="multicluster-engine-console-mce" \
+      maintainer="['acm-component-maintainers@redhat.com']" \
+      description="multicluster-engine-console-mce"
+#     version="${CI_CONTAINER_VERSION}" \
+#     upstream-ref="${CI_CONSOLE_UPSTREAM_COMMIT}" \
+#     upstream-url="${CI_CONSOLE_UPSTREAM_URL}" \
```

Comment:
@gurnben I tried to base this off of downstream as much as possible, as @Kyl-Bempah said it will eventually replace downstream, plus we have an OOM issue with the RHTAP build currently, and the downstream Dockerfile was structured a bit differently with fewer layers... so maybe this will solve it? 

But anyway, does it make sense to be applying these labels, and are there equivalent variables available for `CI_CONTAINER_VERSION`, `CI_CONSOLE_UPSTREAM_COMMIT`, and `CI_CONSOLE_UPSTREAM_URL`?

Reply:
I think there may be some analagous vars - @scoheb are there any env vars in the build context we could leverage?

Reply:
There are no such variables that can be substituted in RHTAP. These were injected into the Dockerfile previously because there was another process with took the Dockerfile as input, filling in the variable templates, and produced the _actual_ Dockerfile. 

Instead, the buildah task will add the following labels at build time: https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah/0.1/buildah.yaml#L154-L160

Reply:
Thanks @arewm @scoheb 

I have removed these.